src/components/Articles/ReadArticle.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import user from '../../assets/images/user.png';
+
+const articleCreated = articleDate => {
```

Comment:
You could use `react-moment` to display date, time or duration

Reply:
Thank you David, sadly `react-moment` does not return the `dateOnly` which is what I desire at this point.